app/workers/classifications_dump_worker.rb

```diff
@@ -8,33 +8,49 @@ class ClassificationsDumpWorker
 
   sidekiq_options queue: :data_high
 
+  attr_accessor :cache
+
   def perform_dump
     CSV.open(csv_file_path, 'wb') do |csv|
-      cache = ClassificationDumpCache.new
+      @cache = ClassificationDumpCache.new
```

Comment:
Maybe just a method `def cache; @cache ||= ClassificationDumpCache.new; end`? I tend to prefer that (sets it up on first use) over instance variables that may or may not be set up.